tutorials/monitoring/1-component-overview/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Monitoring NServiceBus solutions: Component overview"
+reviewed: 2017-10-10
+summary: An overview of all of the Particular Service Platform monitoring tools
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Setting up monitoring environment"
+  nextUrl: tutorials/monitoring/2-setting-up-environment
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This first lesson explains all of the components involved in monitoring an NServiceBus solution with the Particular Service Platform.
+
+## Component overview
+
+```mermaid
+graph TD
+
+subgraph Endpoint
+  nsb(NServiceBus) --> metrics
+  metrics(NServiceBus.Metrics) --> plugin
+
+  plugin(NServiceBus.Metrics.ServiceControl)
+end
+
+plugin --> q(Monitoring Queue)
+
+nsb --> auditQ(Audit Queue)
+nsb --> errorQ(Error Queue)
+
+subgraph ServiceControl
+  scinstance
+  scmonitoring
+end
+
+auditQ --> scinstance(ServiceControl Instance)
+errorQ --> scinstance
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+scinstance --> pulse
+
+```
+
+![the above renders like this](component-overview.png)
+
+Each system being monitored will push data to three main queues:
+
+- Monitoring: Each endpoint will calculate and send metric data to this queue. This data can tell us about the health and performance of an endpoint.
```

Comment:
"gather" instead of "calculate"
"this data holds information about" instead of "this data can tell us"